### PR TITLE
#2471 Added description for support for path variables

### PR DESCRIPTION
**Pull request by neetapimpalnerkar**
Please review #2471.

### CODE REVIEW

File: source_en/Introduction/ChangeLog.rst [hunk 1]
```diff
@@ -752,7 +754,7 @@ Change Log
       - :doc:`../ArchitectureInDetail/DataAccessDetail/DataAccessCommon`
       - Modified in accordance with bug fixes of common library
 
-        * Added description about handling double byte wild card characters (\ ``％``\ , \ ``＿``\)\ , in accordance with bug fixes of common library (\ `terasoluna-gfw#78 <https://github.com/terasolunaorg/terasoluna-gfw/issues/78>`_\ ).
+        * Added description about handling double byte wild card characters (\ ``％``\ , \ ``＿``\ )\ , in accordance with bug fixes of common library (\ `terasoluna-gfw#78 <https://github.com/terasolunaorg/terasoluna-gfw/issues/78>`_\ ).
```
Comment (asaharas) [inline source_en/Introduction/ChangeLog.rst]: please check diff carefully
and prease remove whitespace

File: source_en/Security/Authorization.rst [hunk 2]
```diff
@@ -446,6 +446,60 @@ For main Expression that can be used, refer :ref:`SpringSecurityAuthorizationPol
 
 |
 
+Reference of path variables
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+In Spring Security 4.1 and subsequent versions, a path variable [#fPathVariableDescription]_ can be used while specifying a resource which applies the access policy
```
Comment (asaharas) [inline source_en/Security/Authorization.rst]: @neetapimpalnerkar 
build failed.
please fix like below
... a path variable \ [#fPathVariableDescription]_\ can be used...

Comment (neetapimpalnerkar) [inline source_en/Security/Authorization.rst]: @asaharas I will check this.

File: source_en/Introduction/ChangeLog.rst (hunk 1 above)
Comment (neetapimpalnerkar) [inline source_en/Introduction/ChangeLog.rst]: @asaharas  I will check this.

**Pull request merged**